added info on trusted_issuers.yaml (#1420)
nuts-foundation/nuts-node: docs/pages/deployment/backup-restore.rst

The backup process will only keep a single file per store.
If you want to keep hourly, daily, and weekly backups, you can achieve this with tools like ``rsync`` and ``rsnapshot`` (or others).

Other
=====

> There's one additional piece of data that requires a backup: the list of trusted DIDs.

Reviewer: ```suggestion
Additionaly, the list of trusted VC issuers must be backed up as well. 
```

Trusted issuers of VCs are stored in  ``vcr/trusted_issuers.yaml`` inside the ``datadir`` directory.

> If the contents of this file is your primary store for trusted DIDs, make sure to make a backup.

Reviewer: ```suggestion
If the contents of this file is your primary store for trusted issuers (you're not managing them in an external administrative system), make sure to make a backup.
```

Restore
*******

To restore a backup, follow the following steps:

- shutdown the node.
- remove the following directories from the ``datadir``: ``events``, ``network``, ``vcr`` and ``vdr``

> - follow the restore procedure for your storage (BBolt, Redis, Vault)

Reviewer: ```suggestion
- follow the restore procedure for your storage (BBolt, Redis, Hashicorp Vault)
```

- restore the ``vcr/trusted_issuers.yaml`` file inside ``datadir``.
- start your node
- make an empty POST call to ``/internal/network/v1/reprocess?type=application/vc+json``
- make an empty POST call to ``/internal/network/v1/reprocess?type=application/ld+json;type=revocation``

When making the API calls, make sure you use the proper URL escaping.
Reprocess calls return immediately and will do the work in the background.

BBolt
=====

> In the fourth step copy ``network/data.db``, ``vcr/backup-issued-credentials.db`` and ``vdr/didstore.db`` from your backup to the ``datadir`` (keep directory structure).

Reviewer: ```suggestion
In the fourth step copy ``network/data.db``, ``vcr/backup-issued-credentials.db`` and ``vdr/didstore.db`` from your backup to the ``datadir`` (keep the directory structure).
```
